packages/fiori-generator-shared/src/cap/index.ts

```diff
@@ -43,3 +43,14 @@ export function getCapFolderPathsSync(capProjectPath: string): CapCustomPaths {
 
     return capPaths;
 }
+
+/**
+ * Get the path to the annotations file for a project.
+ *
+ * @param projectName The name of the project.
+ * @param appPath path to the application
+ * @returns {string} The path to the annotations file.
+ */
+export function getAnnotationPath(projectName: string, appPath = 'app'): string {
```

Comment:
Can we leave this with the specific cap code that uses it? No need to reuse since its only used once.

Reply:
moved `getAnnotationPath` to `cap-config-writer`